zigbeetuya/descriptors/_TZE200_ztc6ggyl-TS0601.txt

```diff
@@ -0,0 +1,14 @@
+ D | Zigbee: --> ZigbeeNode(0x7ba5, DC:8E:95:FF:FE:54:99:DD, _TZE200_ztc6ggyl (0x1002), TS0601, Router, RxOn:true)
```

Comment:
You really want to keep those in the source?

Reply:
It happens too often to me that I want to fix/change/add something to a plugin and need to dig endless chat log histories to find the descriptors again... So I figured this makes sense...